hops/featurestore.py

```diff
@@ -2052,7 +2053,7 @@ def import_featuregroup_redshift(storage_connector, query, featuregroup, primary
     fs_utils._log("Feature group imported successfully")
 
 
-def import_featuregroup_s3(storage_connector, path, featuregroup, primary_key=None, description="", featurestore=None,
+def import_featuregroup_s3(storage_connector, path, featuregroup, primary_keys=None, description="", featurestore=None,
```

Comment:
In the other methods you use `[]` as default for `primary_keys`, any reason why not here?

Reply:
thx, fixed